velox/connectors/hive/tests/PartitionIdGeneratorTest.cpp

```diff
@@ -295,4 +289,41 @@ TEST_F(PartitionIdGeneratorTest, supportedPartitionKeyTypes) {
   }
 }
 
+TEST_F(PartitionIdGeneratorTest, nullPartitionValue) {
+  // Test single key
+  PartitionIdGenerator sIdGenerator(ROW({INTEGER()}), {0}, 100, pool(), true);
```

Comment:
these variable names are not very readable; these are essentially single-letter variables + type (s, m), which violate best practices; please, rename

Reply:
good advice,  i will fix this